tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -114,6 +117,15 @@ def test_params_default(self):
                     self.Parameter(name='test', desc='  test description2 \n')]
         self.check_docstring(doc, expected)
 
+    def test_excps_default(self):
```

Comment:
https://github.com/coala/coala/pull/4281/files#diff-603a58b800b18edbad56a01690a21e05R172
here it is.